### ISSUE TITLE
Is it possible to opt-out of object cloning

### ISSUE THREAD

**Issue by adamreisnz** [Alfred-Skyblue/vue-draggable-plus#225]
Currently, cloning of the objects happens in VueDraggable, and this is causing all sorts of problems where we compare objects later down the line, e.g. equality checks no longer work because we're now looking at a clone, and classes are lost etc.

I know there are workarounds, but ideally, I'd like to be able to disable/opt-out of cloning in this library altogether.

Is this possible to opt out of cloning altogether? The online guides are not very clear on this.

**Comment by Alfred-Skyblue** [comment]
Can I understand that you don't want the copy action to take place? This will cause you to lose some properties. If so, you can choose:

```vue
<template>
  <VueDraggable :clone="(element) => element"></VueDraggable>
</template>
```

This way you can avoid cloning, or you can choose to pass the clone function yourself to complete the operation:

```vue
<template>
  <VueDraggable :clone="cloneDeep"></VueDraggable>
</template>

<script setup>
import { cloneDeep } from 'lodash'
</script>
```

**Comment by adamreisnz** [comment]
Yes, the reason the built-in copy/clone action doesn't work, is because it transforms class instances into plain objects, which breaks the app as these objects lose their virtual getters/type checks.

This is more or less how I resolved it yes:

```
:clone="substep => substep.clone()"
```

However, I am surprised that this is the default behaviour of VueDraggable. It seems to be out of scope and lead to unexpected side effects by cloning the objects by default.
